source/extensions/filters/listener/original_dst/original_dst.cc

```diff
@@ -20,12 +22,47 @@ Network::FilterStatus OriginalDstFilter::onAccept(Network::ListenerFilterCallbac
 
   if (socket.addressType() == Network::Address::Type::Ip) {
     Network::Address::InstanceConstSharedPtr original_local_address = getOriginalDst(socket);
-
     // A listener that has the use_original_dst flag set to true can still receive
     // connections that are NOT redirected using iptables. If a connection was not redirected,
     // the address returned by getOriginalDst() matches the local address of the new socket.
     // In this case the listener handles the connection directly and does not hand it off.
     if (original_local_address) {
+#ifdef WIN32
+      // See how to perform bind or connect redirection on MSDN
```

Comment:
on MSDN -> here:

Reply:
fixed in c6718e872a64a57e440c090fa2df9fc6dd5a40eb